### PR TITLE
Streamline revit system test base

### PR DESCRIPTION
**Pull request by ZiyunShang**
### Purpose

Remove GetAllFamilyInstances and GetAllWalls from RevitSystemTestBase class.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@AndyDu1985 

### FYIs

@mjkkirschner @QilongTang

### CODE REVIEW

File: test/Libraries/RevitTestServices/RevitSystemTestBase.cs [hunk 1]
```diff
@@ -363,12 +363,44 @@ protected static void SwapCurrentModel(string modelPath)
...
+            // This is required to pickup certain dlls.
+            var path =
+                Environment.GetEnvironmentVariable(
+                    "Path",
+                    EnvironmentVariableTarget.Process) + ";" + corePath;
+            Environment.SetEnvironmentVariable("Path", path, EnvironmentVariableTarget.Process);
+        }
+    }   
+
+    public class RevitSystemTestHelper
```
Comment (AndyDu1985) [inline test/Libraries/RevitTestServices/RevitSystemTestBase.cs]: Could you move this class to a seprate file?

Comment (ZiyunShang) [reply test/Libraries/RevitTestServices/RevitSystemTestBase.cs]: I have a new commit, could you help review it?

File: test/Libraries/RevitIntegrationTests/BugTests.cs [hunk 2]
```diff
@@ -762,7 +762,7 @@ public void SelectionButtonShouldBeDisabledAfterOpeningNewDocument()
 
         [Test]
         [Category("RegressionTests")]
-        [TestModel(@".\Samples\DynamoSample.rvt")]
+        [TestModel(@".\Samples\DynamoSample_2020.rvt")]
```
Comment (QilongTang) [inline test/Libraries/RevitIntegrationTests/BugTests.cs]: Is this for speeding the test up?

Comment (ZiyunShang) [reply test/Libraries/RevitIntegrationTests/BugTests.cs]: No, I just found that the DynamoSample.rvt file was deleted by last PR https://github.com/DynamoDS/DynamoRevit/pull/2387, and DynamoRevit would get Sample file from nuget.org.
I'm sorry for this missing.

**Pull request merged**